sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -660,12 +660,21 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+
+  val QUERY_EXECUTION_LISTENERS =
+    ConfigBuilder("spark.sql.queryExecutionListeners")
+      .doc("QueryExecutionListeners to be attached to the SparkSession")
```

Comment:
Can you improve this line? Add what you wrote in the `sql-programming-guide.md`?